06-RStudio-VSCode-AI-Integrations.Rmd

```diff
@@ -1,13 +1,70 @@
+---
+editor_options:
+  markdown:
+    wrap: 80
+---
+
 # Using AI Integrations
 
+In the ROSSyndicate, we are actively trying to use some of the large language
+models to make our jobs easier/more efficient. Below is one of the most
+sustainable ways to integrate the CoPilot LLM into your workflow.
+
+We encourage all members of the lab to sign up for (and use!) any of the
+following sites:
+
+-   [ChatGPT](https://chat.openai.com/)
+
+-   GitHub CoPilot
+
+-   [Perplexity AI](https://www.perplexity.ai/)
+
 ## Using GitHub CoPilot in VSCode + RStudio
 
+VSCode also has an extension for GitHub CoPilot, it will auto complete your
+comments and code in nearly any programming language. By having open both VSCode
+and RStudio and enabling auto-save in both programs, you can hop between the two
+programs and code will update between the two as you switch. Big thanks to Sam
+Struthers for figuring out this integrated workflow.
+
 ## Accessing GitHub CoPilot as a student/teacher/researcher
 
+Access to GitHub CoPilot is currently free to students/teachers/researchers.
+Follow the instructions here to submit an application to gain access to the
+program: <https://education.github.com/discount_requests/pack_application>. A
+screenshot of your 'employee information' from the [human resources
+portal](https://aar.is.colostate.edu/service/hrprod) will be sufficient
+information to verify your employment. GitHub usually will give you access to
+the educational portal within 1 business day.
+
 ## Downloading VSCode and installing the GitHub CoPilot extension
 
+Download VSCode here: <https://code.visualstudio.com/>. Set up the GitHub and
+GitHub CoPilot extensions by adding them via the 'Marketplace'. You can bring up
+the Extensions from the menu bar under 'Code', 'Settings', 'Extensions'.
+
+You'll want to install and activate 'GitHub Pull Requests and Issues' as well as
+'GitHub Copilot'. You will be directed to Sign In using your GitHub credentials.
+You may need to elect to 'enable' Copilot.
+
 ## Setting up AutoSave in VSCode + RStudio
 
+In RStudio, open the Preferences pane from the menu bar under 'RStudio', then
+find the following pane:
+
+![](images/r%20autosave.png)
+
+Set up Auto-save and 'Apply'.
```

Comment:
as in, make sure that the "Always save R scripts before sourcing" and "Automatically save when editor loses focus" are selected, like in the image?